Extensions/IISWebAppDeploy/Src/Tasks/IISWebAppDeploy/IISWebAppDeployV2/Main.ps1

```diff
@@ -0,0 +1,50 @@
+[CmdletBinding()]
+param()
+
+$env:CURRENT_TASK_ROOTDIR = Split-Path -Parent $MyInvocation.MyCommand.Path
+
+Import-Module $env:CURRENT_TASK_ROOTDIR\ps_modules\VstsTaskSdk
+
+Trace-VstsEnteringInvocation $MyInvocation
+
+# Import the loc strings.
+Import-VstsLocStrings -LiteralPath $env:CURRENT_TASK_ROOTDIR\task.json
+
+# Get inputs for the task
+$machinesList = Get-VstsInput -Name MachinesList -Require
+$adminUserName = Get-VstsInput -Name AdminUserName -Require
+$adminPassword = Get-VstsInput -Name AdminPassword -Require
+$winrmProtocol = Get-VstsInput -Name WinrmProtocol -Require
+$testCertificate = Get-VstsInput -Name TestCertificate
+$webDeployPackage = Get-VstsInput -Name WebDeployPackage -Require
+$webDeployParamFile = Get-VstsInput -Name WebDeployParamFile
+$overRideParams = Get-VstsInput -Name OverRideParams
+$websiteName = Get-VstsInput -Name WebsiteName -Require
+$removeAdditionalFiles = Get-VstsInput -Name RemoveAdditionalFiles
+$excludeFilesFromAppData = Get-VstsInput -Name ExcludeFilesFromAppData
+$takeAppOffline = Get-VstsInput -Name TakeAppOffline
```

Comment:
You can pass the argument ` -AsBool` which ensures only boolean input is provided.

Reply:
It is done